pkg/controller/nodelifecycle/node_lifecycle_controller.go

```diff
@@ -1333,15 +1327,15 @@ func (nc *Controller) processPod(ctx context.Context, podItem podUpdateItem) {
 	// Pods are processed by TaintManager.
 	if !nc.runTaintManager {
 		if err := nc.processNoTaintBaseEviction(ctx, node, currentReadyCondition, nc.nodeMonitorGracePeriod, pods); err != nil {
-			klog.Warningf("Unable to process pod %+v eviction from node %v: %v.", podItem, nodeName, err)
+			klog.InfoS("Unable to process pod eviction from node", "pod", klog.KRef(podItem.namespace, podItem.name), "nodeName", nodeName, "err", err)
 			nc.podUpdateQueue.AddRateLimited(podItem)
 			return
 		}
 	}
 
 	if currentReadyCondition.Status != v1.ConditionTrue {
 		if err := controllerutil.MarkPodsNotReady(ctx, nc.kubeClient, nc.recorder, pods, nodeName); err != nil {
-			klog.Warningf("Unable to mark pod %+v NotReady on node %v: %v.", podItem, nodeName, err)
+			klog.InfoS("Unable to mark pod NotReady on node", "pod", klog.KRef(podItem.namespace, podItem.name), "nodeName", nodeName, "err", err)
```

Comment:
I think Warning should be promoted to Error..

Reply:
The recommendation is to treat it like an information message: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#change-log-functions

Reply:
agree